packages/overlays/addon/components/drawer/index.hbs

```diff
@@ -1,6 +1,7 @@
 <Overlay
   @isOpen={{@isOpen}}
   @onClose={{this.handleClose}}
+  @onOpen={{this.handleOpen}}
```

Comment:
We can simplify and just pass the argument directly:

```suggestion
  @onOpen={{@onOpen}}
```

This is possible because the underlying component checks if the argument exists and is a function.

Reply:
Thats a great point. Would it make sense to do the same for `this.handleClose`?  
like:
`@onClose={{onClose}}`